DNA_RNA_aminoacid_tools.py

```diff
@@ -0,0 +1,145 @@
+import modules.fastq_filtering as ff
+import modules.aminoacid_functions as af
+import modules.dna_rna_functions as drf
+
+def filter_seqs(input_path: str, gc_bounds: tuple, length_bounds: tuple, quality_threshold = 0, output_filename = ''):
```

Comment:
```suggestion
def filter_fastq(input_path: str, gc_bounds: tuple, length_bounds: tuple, quality_threshold = 0, output_filename = ''):
```
Все  таки мы же тут с конкретным типом данных работаем